apps/webClient/src/components/View/index.tsx

```diff
@@ -24,31 +39,60 @@ const useStyles = makeStyles(theme => ({
   main: {
     width: "100%",
     flexGrow: 1,
-    overflowY: "auto",
-    overflowX: "scroll",
   },
-  pageWrapper: {
-    backgroundColor: LIGHT_BLUE,
-    background: GRADIENT,
-    height: "100%",
+  redPageWrapper: {
+    backgroundColor: RED,
+    background: RED_GRADIENT,
   },
-  pageContainer: {
-    padding: 0,
+  greenPageWrapper: {
+    backgroundColor: GREEN,
+    background: GREEN_GRADIENT,
+  },
+  bluePageWrapper: {
+    backgroundColor: BLUE,
+    background: BLUE_GRADIENT,
+  },
+  yellowPageWrapper: {
+    backgroundColor: YELLOW,
+    background: YELLOW_GRADIENT,
   },
 }));
 
 const View: React.FC<Props> = ({ children }) => {
   const classes = useStyles();
+  const { pathname } = useLocation();
+
+  const derivePageRootBackgroundColor = (path: string) => {
+    let coloredPageRoot = [classes.root];
```

Comment:
we can change later, but this should be `const` -- you can append to an array and it doesn't change the array.